sharing_portal/templates/sharing_portal/request_day_pass.html

```diff
@@ -0,0 +1,67 @@
+{% extends 'layout/default.html' %}
+{% load i18n cms_tags static pipeline bootstrap3 %}
+{% load render_bundle from webpack_loader %}
+
+{% block title %}Request Day Pass | Chameleon Cloud{% endblock %}
+
+{% block styles %}
+{{ block.super }}
+<link rel="stylesheet" type="text/css" href="{% static 'sharing_portal/artifacts.css' %}">
+{% endblock %}
+
+{% block content %}
+{# Experiments / [current_exp_title] subheader #}
+<ol class="breadcrumb">
+  <li>
+    <a href="{% url 'sharing_portal:index_all' %}">All</a>
+  </li>
+  <li>
+    <a href="{% url 'sharing_portal:detail' artifact.pk %}">{{ artifact.title }}</a>
+  </li>
+  <li class="active">Request Day Pass</li>
+</ol>
+
+<div>
+
+  <form id="artifactShareForm" method="post">
+    {% csrf_token %}
+
+    <div class="layoutGrid">
+      <div class="layoutGrid__main">
+        <div class="artifactDetail">
+          <header class="artifactDetail__heading">
+            <h2 class="artifactTitle">Request Day Pass</h2>
+          </header>
+            <p>
+            You are requesting to temporarily have access to Chameleon in
+            order to reproduce {{ artifact.title }}. By submitting this form,
+            an email will be sent to the PI of the project. If you have any
+            information that will convince them to approve your request, please
```

Comment:
maybe instead of "that will convince them to approve your request" say "relevant to your request", to make it sound less like this is their opportunity to crack open their copy of _How to Win Friends and Influence People_ :D